homeassistant/components/overkiz/sensor.py

```diff
@@ -359,6 +359,13 @@ class OverkizSensorDescription(SensorEntityDescription):
         key=OverkizState.IO_ELECTRIC_BOOSTER_OPERATING_TIME,
         name="Electric Booster Operating Time",
     ),
+    # Cover
+    OverkizSensorDescription(
+        key=OverkizState.CORE_TARGET_CLOSURE,
+        name="Target Closure",
+        native_unit_of_measurement=PERCENTAGE,
+        entity_registry_enabled_default=False,
```

Comment:
Disabled by default since this sensor is only useful for users with 'Uno devices'. These are devices that cannot broadcast their current status, but where the device will know what the target closure percentage is.
We don't use this state to show the actual position in cover, since we don't implement optimistic mode.